roles/monitoring/tasks/main.yml

```diff
@@ -0,0 +1,60 @@
+- name: Enable extras repo
+  command: subscription-manager repos --enable=rhel-7-server-extras-rpms
+  when: inventory_hostname in groups.loadbalancers
+- name: Install docker
+  yum:
+    name: docker
+    state: latest
+  when: inventory_hostname == 'localhost' or inventory_hostname in groups.loadbalancers
+  become: yes
+  become_method: sudo
+- name: Enable and start docker daemon service
+  systemd:
+    name: docker
+    state: started
+    enabled: yes
+    daemon_reload: yes
+  when: inventory_hostname == 'localhost' or inventory_hostname in groups.loadbalancers
+  become: yes
+  become_method: sudo
+- name: create persistent storage volume for zabbix
+  command: docker run -d -v /var/lib/mysql --name zabbix-db-storage busybox:latest
+  when: inventory_hostname == 'localhost'
+  become: yes
+  become_method: sudo
+  args:
+    creates: .zabbix_persistence_created 
+- name: install zabbix db
+  command: docker run -d --name zabbix-db -v /backups:/backups -v /etc/localtime:/etc/localtime:ro --volumes-from zabbix-db-storage --env="MARIADB_USER=zabbix" --env="MARIADB_PASS=my_password" monitoringartist/zabbix-db-mariadb
+  when: inventory_hostname == 'localhost'
+  become: yes
+  become_method: sudo
+  args:
+    creates: .zabbix_db_created 
+- name: install zabbix server
+  command: docker run -d --name zabbix -p 80:80 -p 10051:10051 -v /etc/localtime:/etc/localtime:ro --link zabbix-db:zabbix.db --env="ZS_DBHost=zabbix.db" --env="ZS_DBUser=zabbix" --env="ZS_DBPassword=my_password" monitoringartist/zabbix-xxl:latest
```

Comment:
Shouldn't we be using a module to interface with docker e.g. http://docs.ansible.com/ansible/latest/docker_container_module.html#docker-container

Reply:
Yes - again I'd like to add that to the todo list rather than block this merge with it.